### PR TITLE
Sync dev with master

### PR DESCRIPTION
**Pull request by abhi18av**
* adding resource and readme

* fix typo

* adding credits for the original manual

* removing author names

Co-authored-by: Abhinav Sharma <abhi18av@users.noreply.github.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Mtbseq-nf
+
+OBS: This still a Work in progress!
```
Comment (abhi18av) [inline README.md:3]: It's more common to use `NOTE` rather than `OBS` (observation)

```suggestion
**NOTE**: This still a Work in progress!
```

**Comment by abhi18av** [comment]
Marking this PR as ready to review and releasing the workflow to master as an alpha version.

**Comment by Mxrcon** [comment]
ok We should try it and them try provide some fix list for the issues

**Comment by abhi18av** [comment]
I think that we can probably push another round with `stubs` and then let's switch to real data once we accomodate the NCBI task 
#22

**Comment by Mxrcon** [comment]
ok! I'll test that on our tower if there's no problem for you

**Pull request merged**